releases/manifest/dev/firmware_CCP_ESP32.json

```diff
@@ -0,0 +1 @@
+{"type": "CCP_ESP32", "version": "100", "host": "influxdb.canair.io", "port": 8080, "bin": "/releases/dev/canairio_CCP_ESP32_rev100.bin"}
```

Comment:
these files, manifest* files, should be auto generated. Please remove these files from the PR and from your repo.